packages/runtime/src/helpers/verification.ts

```diff
@@ -105,6 +107,15 @@ export const checkForRootPublish = ({
 }
 
 export const checkZipSize = async (file: string, maxSize: number = LAMBDA_MAX_SIZE): Promise<void> => {
+  // Requires contacting the Netlify Support team to fully enable.
+  // Enabling this without contacting them can result in failed deploys.
+  if (isBundleSizeCheckDisabled()) {
+    console.warn(
+      'Function bundle size check was DISABLED with the DISABLE_BUNDLE_ZIP_SIZE_CHECK environment. Your deployment will break if it exceeds the maximum supported size of function zip files in your account.',
```

Comment:
I believe you missed a word?

```suggestion
      'Function bundle size check was DISABLED with the DISABLE_BUNDLE_ZIP_SIZE_CHECK environment variable. Your deployment will break if it exceeds the maximum supported size of function zip files in your account.',
```

Reply:
Good catch!